### PR TITLE
UI guidelines: Button fixes

### PR DESCRIPTION
**Pull request by dpobel**
* Change `.ez-button` to be neutral by default
* nitpick: Inverted button background colors so that the less used being the alternative one

### CODE REVIEW

**Comment by dpobel** [comment]
manual ping to @inakijv, you does not seem to be Reviewers list :)

File: ui-guidelines/index.html [hunk 1]
```diff
@@ -162,27 +162,27 @@ <h4 class="u-move-down-xlarge">Basic buttons</h4>
...
+										<p class="button-card-header">Secondary button</p>
 										<div class="button-card-content">
-											<button type="button" class="ez-button ez-button-neutral">Cancel</button>
+											<button type="button" class="ez-button ez-button-secondary">Save</button>
 										</div>
 									</div>
 									<div class="button-code">
 <pre>
-<code class="language-markup"><script type="prism-html-markup"><button type="button" class="ez-button ez-button-neutral">Cancel</button></script></code>
+<code class="language-markup"><script type="prism-html-markup"><button type="button" class="ez-button ez-button-secondary">Save</button></script></code>
```
Comment (SylvainGuittard) [inline ui-guidelines/index.html]: I totally understand the approach of converting `ez-neutral` into `ez-button` -> +1
But we have to include the default `ez-button` in the guidelines.

**Comment by SylvainGuittard** [comment]
I know that Iñaki is also working on some improvements regarding your feedbacks. I will let @inakijv decide how he wants to deal with this PR.

File: ui-guidelines/index.html (hunk 1 above)
Comment (dpobel) [inline ui-guidelines/index.html]: it's already done in the guideline
ui

**Comment by SylvainGuittard** [comment]
Thanks @dpobel for the feedback and the contribution. 
Recommendations integrated in #46

**Pull request closed without merging**